sandboxed_containers/troubleshooting-sandboxed-containers.adoc

```diff
@@ -17,9 +17,9 @@ include::modules/about-must-gather.adoc[leveloffset=+2]
 
 To collect {sandboxed-containers-first} data with `must-gather`, you must specify the
 {sandboxed-containers-first} image:
-[source,terminal]
+[source,terminal,subs="attributes+"]
 ----
---image=registry.redhat.io/openshift-sandboxed-containers/osc-must-gather-rhel8:1.2.0
+--image=registry.redhat.io/openshift-sandboxed-containers/osc-must-gather-rhel8:{sandboxed-containers-version}.0
```

Comment:
Should this be sandboxed-containers-version-z?

Reply:
Same comment as above.